frontend-react/src/components/FileHandlers/FileHandlerSchemaSelectionStep.tsx

```diff
@@ -62,7 +62,7 @@ export default function FileHandlerSchemaSelectionStep({
                         - Select -
                     </option>
                     {schemaOptions.map(({ title, value }) => (
-                        <option key={value} value={value}>
+                        <option key={`key-${value}`} value={value}>
```

Comment:
Any particular reason for this change? Being nitpicky but `key` is purely a react technical thing so unless we're adding some randomness to it manually, dressing it up with a static string template functionally changes nothing.

Reply:
That is true, this is because React complains that the `key` property _must_ be unique, so without that prefix, it'll give us that warning endlessly.

Reply:
It stops warning with a string that doesn't change the uniqueness? Weird. I would add the index from the map iteration to the string just to be safe.

Reply:
Agree, key should be index of the map

Reply:
Yup, totally makes sense.